data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -167,40 +193,37 @@ def _climate_factor(self) -> bool:
             )
```

Comment:
We still have duplicate code here – are you planning to remove that as part of this PR and just calculate the same values once?